app/install/microsoft-azure.md

```diff
@@ -0,0 +1,99 @@
+---
+id: page-install-method
+title: Install - Microsoft Azure
+header_title: Microsoft Azure
+header_icon: /assets/images/icons/icn-installation.svg
+breadcrumbs:
+  Installation: /install
+---
+
+## Table of Contents
+
+<!-- FIXME the list below should be an unordered list, but currently those do not render correctly in this section of the Docs site - depends on https://github.com/Kong/docs.konghq.com/issues/917 -->
+1. [Running Kong on Azure Container Instances](#running-kong-on-azure-container-instances)
+1. [Running PostgreSQL on Azure with Azure Database for PostgreSQL](#running-postgresql-on-azure-with-azure-database-for-postgresql)
+1. [Running Cassandra on Azure with Azure Cosmos Db](#running-cassandra-on-azure-with-azure-cosmos-db)
+
+# Running Kong on Azure Container Instances
```

Comment:
Should be `##` - `title: Install - Microsoft Azure` in the front matter is rendered as the page title and the sole H1 header on the page - all other headers must start at H2, to avoid having more than H1 on the page.